src/factories/workflow-specific/Restricted_PIM_Factory_v1.sol

```diff
@@ -0,0 +1,238 @@
+        orchestratorFactory = _orchestratorFactory;
+    }
+
+    //--------------------------------------------------------------------------
+    // Public Mutating Functions
+
+    /// @inheritdoc IRestricted_PIM_Factory_v1
+    function createPIMWorkflow(
+        IOrchestratorFactory_v1.WorkflowConfig memory workflowConfig,
+        IOrchestratorFactory_v1.ModuleConfig memory fundingManagerConfig,
+        IOrchestratorFactory_v1.ModuleConfig memory authorizerConfig,
+        IOrchestratorFactory_v1.ModuleConfig memory paymentProcessorConfig,
+        IOrchestratorFactory_v1.ModuleConfig[] memory moduleConfigs,
+        IBondingCurveBase_v1.IssuanceToken memory issuanceTokenParams
+    ) external returns (IOrchestrator_v1 orchestrator) {
+        // deploy issuance token
+        ERC20Issuance_v1 issuanceToken = new ERC20Issuance_v1(
+            issuanceTokenParams.name,
+            issuanceTokenParams.symbol,
+            issuanceTokenParams.decimals,
+            issuanceTokenParams.maxSupply,
+            address(this) // assigns owner role to itself initially to manage minting rights temporarily
+        );
+
+        // MODIFY AUTHORIZER CONFIG
+        // decode configData of authorizer
+        // set (own) factory as orchestrator admin
+        bytes memory auhorizerConfigData = authorizerConfig.configData;
+        (address realAdmin) = abi.decode(auhorizerConfigData, (address));
+        auhorizerConfigData = abi.encode(address(this));
+        authorizerConfig.configData = auhorizerConfigData;
+
+        // MODIFY FUNDING MANAGER CONFIG
+        // decode configData of fundingManager
+        // set newly deployed token as issuance token
+        bytes memory fundingManagerConfigData = fundingManagerConfig.configData;
+        (
+            ,
+            IFM_BC_Bancor_Redeeming_VirtualSupply_v1.BondingCurveProperties
+                memory bcProperties,
+            address collateralToken
+        ) = abi.decode(
+            fundingManagerConfigData,
+            (
+                address,
+                IFM_BC_Bancor_Redeeming_VirtualSupply_v1.BondingCurveProperties,
+                address
+            )
+        );
+        fundingManagerConfigData =
+            abi.encode(address(issuanceToken), bcProperties, collateralToken);
+        fundingManagerConfig.configData = fundingManagerConfigData;
+        orchestrator = IOrchestratorFactory_v1(orchestratorFactory)
+            .createOrchestrator(
+            workflowConfig,
+            fundingManagerConfig,
+            authorizerConfig,
+            paymentProcessorConfig,
+            moduleConfigs
+        );
+
+        // get bonding curve / funding manager
+        address fundingManager = address(orchestrator.fundingManager());
+
+        // enable bonding curve to mint issuance token
+        issuanceToken.setMinter(fundingManager, true);
+
+        // transfer initial collateral supply from msg.sender to  bonding curve and mint issuance token to recipient
```

Comment:
Double space between `to` and `bonding curve`.

```suggestion
        // Transfer initial collateral supply from msg.sender to bonding curve and mint issuance token to recipient
```

Reply:
Just out of curiosity, @marvinkruse : Do you have some kind of plugin that highlights these kind of style mistakes? I would never have noticed that double space :rofl:

Reply:
I just removed the whitespace but didn't capitalize the first letter for consistency :P